README.md

```diff
@@ -149,14 +149,17 @@ The `coro::generator<T>` construct is a coroutine which can generate one or more
 
 int main()
 {
-    auto task = [](uint64_t count_to) -> coro::task<void> {
+    auto task = [](uint64_t count_to) -> coro::task<void>
```

Comment:
The root README.md is generated by a pre commit git hook with the .githooks/readme-template.md file.  I don't see that file edited here so these updates will get blown away if someone using the githook to commit makes a PR. Can you edit that file and use the githook to properly gen the root README.md please?

Reply:
Yes, the readme-template.md is not edited, but I also didn't remember that I changed the readme.md. I thought it might be auto generated because I changed the `examples/coro_generator.cpp`

Reply:
ah yes, my mistake here, its pulled directly from the example file so this should be good